modules/flowable-process-validation/src/main/java/org/flowable/validation/validator/impl/ExternalInvocationTaskValidator.java

```diff
@@ -49,7 +55,7 @@ protected void validateFieldDeclarationsForEmail(org.flowable.bpmn.model.Process
             }
         }
 
-        if (!toDefined) {
+        if (!recipientDefined) {
```

Comment:
Can we add some tests for this validation as well? e.g. a mail task with only `cc`, another one with only `bcc`